### DISCUSSION TITLE
Nightly builds versioning

### DISCUSSION THREAD

**Discussion by Michael21107** [scripthookvdotnet/scripthookvdotnet#1508] [Q&A] [answered]
since nightly builds are essentially pre-releases of the next version, shouldn't they be named according to the next version (eg. 3.7.0-nightly.2424 or 4.0.0-nightly325)?

**Comment by kagikn** [comment, marked as answer]
well, i didn't care about what [Semantic Versioning 2.0.0](https://semver.org/) define on pre-release versions, but your suggestion makes sense. will change how to calculate nightly versions soon.

**Reply by Michael21107** [reply]
gotcha, thanks a lot

**Reply by kagikn** [reply]
hey, just to let you know, changed how to version nightly-releases after i merged #1519 (starting from the release [v3.7.0-nightly.0](https://github.com/scripthookvdotnet/scripthookvdotnet-nightly/releases/tag/v3.7.0-nightly.0))

**Reply by Michael21107** [reply]
hey, thanks for the info, looks amazing :)
